exp/api/v1beta1/gcpmanagedcontrolplane_types.go

```diff
@@ -147,7 +147,14 @@ type GCPManagedControlPlaneSpec struct {
 	// If not specified, the default version currently supported by GKE will be
 	// used.
 	// +optional
+	//
+	// Deprecated: This field will soon be removed and you are expected to use Version instead.
```

Comment:
I think the optional stylistically (and maybe even from an implementation standpoint) goes next to the field, whereas the Deprecated can also go above in the godoc comment.
Better be safe, example here:
https://github.com/kubernetes-sigs/cluster-api/blob/7a71f4b0ed71cb92fa751e48057df6bb672319bb/api/v1beta1/cluster_types.go#L533-L539